src/runners/cairoRunner.ts

```diff
@@ -30,14 +31,23 @@ export class CairoRunner {
     this.program = program;
     const mainId = program.identifiers.get('__main__.main');
     const mainOffset = mainId !== undefined ? mainId.pc ?? 0 : 0;
+    const builtins = program.builtins;
 
     this.vm = new VirtualMachine();
     this.programBase = this.vm.memory.addSegment();
     this.executionBase = this.vm.memory.addSegment();
 
+    const builtin_stack = builtins.map((builtin) => {
+      switch (builtin) {
+        case 'bitwise':
+          return this.vm.memory.addSegment(new Bitwise());
+        default:
+          throw new Error();
+      }
+    });
     const returnFp = this.vm.memory.addSegment();
     this.finalPc = this.vm.memory.addSegment();
-    const stack = [returnFp, this.finalPc];
+    const stack = [...builtin_stack, returnFp, this.finalPc];
```

Comment:
builtins have an initial_stack method, they may not all just add a pointer in the stack (see python VM)